### ISSUE TITLE
[FEATURE] More options in SuperKnockback 

### ISSUE THREAD

**Issue by Anonimouze** [CCBlueX/LiquidBounce#2601]
### LiquidBounce Branch

Nextgen

### Describe your feature request.

1. OnlyGround: Only sends knockback packets when you are on the ground and not in the air.
2. OnlyMove: Only sends knockback packets when you move and does not send knockback packets when you stand still.
3. OnlyMoveForward: Only sends knockback packets when moving forward and not in any other directions. This option will 
                                   be inside the OnlyMove settings and should only take effect when OnlyMove settings is turned on.

### Additional context

This options would be extremely useful in many situations and bring more customizability to the module. Please do not close this without reading.

**Comment by superblaubeere27** [comment]
> This options would be extremely useful in many situations and bring more customizability to the module.
> 

Ok, but in what situations?

**Comment by 1zun4** [comment]
I feel like you do not really know how SuperKnockback works. It requires being able to sprint. All options you mentioned make SuperKnockback practically useless.

**Comment by Anonimouze** [comment]
> I feel like you do not really know how SuperKnockback works. It requires being able to sprint. All options you mentioned make SuperKnockback practically useless.

Well I think you have misunderstood what I have said in this suggestion. In this request I have never said to disable sprint in any of the points in this request. For further clarification, most of the requests I have made here is intended for the 'Packet' mode in Superknockback, in the 1st point of this request I have suggested to add a feature in this module which would NOT stop the client from sprinting and make superknockback aim for a knockback attack only when on ground, this feature could be useful in all of the modes as turning this option on would make it possible to have more chances of getting a critical hit on a 1.9+ server as the client currently reduces the chances of getting criticals on a 1.9+ server significantly when superknockback module is turned on and there is no settings to stop this from occurring. Similarly in the 2nd point, I have suggested the addition of a feature in this module which would make superknockback aim for a knockback hit only when you are moving and NOT stop the client from sprinting. This feature is already present in in the 'WTap' and 'SprintTap' modes in this module. Finally in the 3rd point, I have suggested the implementation of a feature that would work together with the 2nd feature to make sure the client only aims for a knockback hit when moving on the forward direction.
